download/index.html

```diff
@@ -8,12 +8,11 @@
             <h1 class="center title-header-text">
                 Download
             </h1>
-            <a href="/mediafire/" id="patcher_dl" class="download center center-text small-gap">
-                Download Beyond Melee Windows Patcher
-            </a>
+            <a href="" id="patcher_dl" class="download center center-text small-gap"></a>
             <!-- <a href="/mega/" id="patcher_dl" class="download-small center center-text">
                 Mega Mirror Download Link
             </a> -->
+            <script>const osStr=navigator.platform.toLowerCase();const dlElement=document.getElementById("patcher_dl");if(osStr.includes("mac")){var txt="Install Beyond Melee On macOS";var lnk="#macdl"}else if(osStr.includes("linux")){var txt="Install Beyond Melee On Linux";var lnk="#linuxdl"}else{var txt="Download Beyond Melee Windows Patcher";var lnk="/mediafire/"}dlElement.innerHTML=txt;dlElement.setAttribute("href",lnk);</script>
```

Comment:
Platform detection for download page.

Reply:
the rest of the page isn’t minified, and it’ll be gzipped - it’s probably fine to not squish this all into one line

Reply:
will this run on gh pages? i thought scripts were blocked?

Reply:
Pages doesn't do server-side things (PHP, Node, Java, etc.). JS works fine, it's all in the browser.